src/queries.js

```diff
@@ -16,3 +16,21 @@ export const suggestedKonnectorsConn = {
   as: 'app-suggestions',
   fetchPolicy: defaultFetchPolicy
 }
+
+export const mkHomeMagicFolderConn = t => {
+  return {
+    query: Q('io.cozy.files').where({ path: t('home_config_magic_folder') }),
+    as: 'home-magic-folder',
+    fetchPolicy: defaultFetchPolicy
+  }
+}
+
+export const mkHomeShorcutsConn = folderId => {
+  return {
+    query: Q('io.cozy.files')
+      .where({ dir_id: folderId, class: 'shortcut' })
+      .indexFields(['dir_id', 'class']),
+    as: 'home-shortcuts',
```

Comment:
`dir_id` should be within the name

Reply:
Except this one, it's 👍 for the PR

Reply:
The folder is the one for shortcuts and "shortcuts" is already in the name. I don't think we have a case where there are several potentially conflicting "shortcuts" queries, what problem do you see ?

Reply:
If we are sure that we can only have one `folderId` so I'm ok with that name even if I prefer to have the variable value in the name since, to me, it's rare to only have one value. I think it's better to get / gain the right habit. But I've maybe too biais since Drive is only about the same query run with only one arg change.

Reply:
I agree with the right habit, I've made the change in https://github.com/cozy/cozy-home/pull/1581/commits/a383142463072f34064bc369df950906d80d3f4f, tell me what you think.